cmd/helm/install_test.go

```diff
@@ -23,6 +23,12 @@ import (
 
 func TestInstall(t *testing.T) {
 	tests := []cmdTestCase{
+		// Install chart with repoURL
+		{
+			name:   "install chart with only repoURL",
+			cmd:    "install traefik --repo https://helm.traefik.io/traefik --version 9.18.0",
```

Comment:
I think this piece should use a mock request, but it looks like it requires a lot of code changes. Is there any suitable solution? thank you
@mattfarina